mobify/progressive-web-scaffold
# Update logger.js

## Changes
- Currently, the preview link we generate defaults to "This Tab". But the downside to limiting it to single tab is that when you preview and cross between HTTP and HTTPS, you have to re-preview. "All Tabs" solves this problem. (the background reason is that "This Tab" uses LocalStorage to store the bundle cookie, which isn't shared across HTTP and HTTPS, where as "All Tabs" stores the bundle string in a cookie).

## How to test-drive this PR
- Run `npm run dev`, copy the preview URL, make sure it defaults to "All Tabs" by looking at which radio button is selected by default.
